bee-network/bee-autopeering/src/delay.rs

```diff
@@ -0,0 +1,236 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use rand::{thread_rng, Rng as _};
```

Comment:
Why as _ ?

Reply:
Because we never need to explicitly mention it. We just want to call the method/s it provides.

Reply:
Ah alright, I would have assumed that was enough to not raise a warning

Reply:
Hmm I did a test and got no warning...

Reply:
If it's confusing to you I can remove the underscore. It's something I wanted to try with this crate. For me it's helpful when I read the file again, but I can see that people who only use it when they would introduce or import another type with the same name could get confused. 

The way it is now I could do that (introduce my own type `Rng`) without changing/fixing another line. So in that sense it's more friendly to code changes as I am humbly not blocking a symbol that I don't explicitly mention in the code.

Reply:
this module has been temporarily removed until it's actually being used, so we can delay that discussion for the time being.